example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -171,14 +176,24 @@ public void anr(View view) throws InterruptedException {
         Thread.sleep(anrTimeout + 2000);
     }
 
-    public void enableBreadcrumbs(View view) {
+    public void enableBreadcrumbs(View view) throws Exception {
+        Context appContext = view.getContext().getApplicationContext();
+        if (backtraceClient == null) {
+            throw new Exception("BacktraceClient is null");
```

Comment:
I think it would be better to write a message like "Backtrace integration is not initialized"

Reply:
done